### PR TITLE
common: add aeroPAX Smart Battery

### PR DESCRIPTION
**Pull request by mirkix**
Add aeroPAX Smart Battery

### CODE REVIEW

File: common/source/docs/common-aerotate-dronecan-battery.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- Set :ref:`BATT_MONITOR<BATT_MONITOR>` to 8 (DroneCAN)
+- Set :ref:`BATT_SERIAL_NUM<BATT_SERIAL_NUM>` to the address you have set at the battery adapter. On each battery adapter, 4 bits can be set to encode this field resulting in 16 possible addresses, where addresses 0 and 15 are reserved and cannot be used. The DroneCAN parameter is battery_id.
+
+The aeroPAX batteries can be setup in parallel and series, but max 2 in series. For parallel setups, a protected output is strongly recommended. It enables hot-swap-ability and protects against user errors.
+For parallel setups an additional battery monitor can be added as :ref:`BATT_SUM_MASK<BATT_SUM_MASK>`, this will sum up the currents and average the voltages, for a better overview.
+
+Safety
+======
+
+Using the preflight checklist can enhance your safety, by checking that the state of charge (SOC) is above a specific limit.
```
Comment (Hwurzburg) [inline common/source/docs/common-aerotate-dronecan-battery.rst:62]:
```suggestion
Using Mission Planner's preflight checklist can enhance your safety, by checking that the state of charge (SOC) is above a specific limit.
```

File: common/source/docs/common-aerotate-dronecan-battery.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.. _common-aerotate-dronecan-battery:
+
+===========================
+Aerotate DroneCAN Battery
+===========================
```
Comment (Hwurzburg) [inline common/source/docs/common-aerotate-dronecan-battery.rst:5]: overline/underline must be same length as heading text

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
A couple of nits...looks good thanks

File: common/source/docs/common-aerotate-dronecan-battery.rst (hunk 2 above)
Comment (mirkix) [reply common/source/docs/common-aerotate-dronecan-battery.rst]: Done

**Review by Hwurzburg — APPROVED** [review]
LGTM...thanks

**Pull request merged**